app/controllers/sessions_controller.rb

```diff
@@ -4,9 +4,10 @@ def new
   end
 
   def create
-    user = User.find_by email: params[:session][:email].downcase
+    user = find_user
```

Comment:
`find_user` em gọi sử dụng `before_action` nhé
để line này ko cần thiết nhé

Reply:
cái này nếu dùng before_action thì thành ra phải đổi user đó thành @user à chị? vì thực ra cái hàm này em tách ra ngoài vì rubocop nó bảo hàm create to quá nên em chữa cháy bằng cách như vậy :v

Reply:
dùng @user đi em